### PR TITLE
deps.nltk.sh: Add nltk_data installation in coala

### PR DESCRIPTION
**Pull request by tushar-rishav**
This commit fetches `nltk_data` so that imperative
mood test from `GitCommitBear` would work for coala.

Fixes https://github.com/coala-analyzer/coala/issues/2011

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,3 +4,6 @@ setuptools>=19.2
 libclang-py3==0.2
 validators~=0.10
 appdirs~=1.4.0
+# Stable version 3.2 is problematic in Windows.
+# Use >=3.3 once nltk-3.3 is released.
+nltk~=3.1.0
```
Comment (Makman2) [inline requirements.txt]: nope that one doesn't belong here :D
isn't it installed automatically anyway with `coala-bears`?

**Comment by Makman2** [comment]
ack 7fd80d9

**Comment by Makman2** [comment]
@rultor merge

**Comment by tushar-rishav** [comment]
@rultor merge

**Pull request merged**